src/main/java/org/jboss/arquillian/testcontainers/TestContainerInstances.java

```diff
@@ -0,0 +1,46 @@
+package org.jboss.arquillian.testcontainers;
+
+import java.util.List;
+
+import org.testcontainers.containers.GenericContainer;
+
+public class TestContainerInstances {
```

Comment:
Minor comment that we could do in a follow up, but maybe we should make this implement `Iterable<GenericContainer<?>>`.

Reply:
Done